test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -1675,6 +1675,9 @@
                 "imaging_protocols": [],
                 "library_preparation_protocols": [
                     {
+                        "document_id": [
```

Comment:
I don't think this should be a list in contributions. Each dict in the outer list represents a single protocol and one protocol can only have one ID. Correct me if I am wrong but the other inner entities in contributions don't have lists for document_id either.

Reply:
In an aggregate document all inner entities (except `files` entities in a `bundles` or `files` aggregate document) either don't include the `document_id` field, or it is a list.
Examples:
- [Sequencing process inner entity in bundles aggregate ](https://github.com/DataBiosphere/azul/blob/19fdbcf8a5ffc6b8e22d2077a14f1f408003510c/test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json#L1506)
- [Specimens inner entity in bundles aggregate](https://github.com/DataBiosphere/azul/blob/19fdbcf8a5ffc6b8e22d2077a14f1f408003510c/test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json#L1514)
- [Donor inner entity in files aggregate](https://github.com/DataBiosphere/azul/blob/19fdbcf8a5ffc6b8e22d2077a14f1f408003510c/test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json#L2234)
- [Cell suspension inner entity in samples aggregate](https://github.com/DataBiosphere/azul/blob/19fdbcf8a5ffc6b8e22d2077a14f1f408003510c/test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json#L2565)

Reply:
@hannes-ucsc : "Apparently I mistook this occurrence as being part of the contribution. Daniel and I went through this and did not find any such occurrence. The only occurrences of arrays in `document_id` are in aggregates."